crates/bevy_ui/src/geometry.rs

```diff
@@ -318,6 +268,15 @@ impl UiRect {
             ..Default::default()
         }
     }
+
+    /// Alias of [`UiRect::UNDEFINED`]
+    pub const DEFAULT: UiRect = UiRect::UNDEFINED;
+
+    /// Creates a [`UiRect`] where all sides are [`Val::Undefined`]
+    pub const UNDEFINED: UiRect = UiRect::all(Val::Undefined);
+
+    /// Creates a [`UiRect`] where all sides are [`Val::Auto`]
+    pub const AUTO: UiRect = UiRect::all(Val::Auto);
```

Comment:
Do we have it documented somewhere what the difference between undefined and auto is?

Reply:
@TimJentzsch In general the meaning of `Auto` depends on the property being set. For a `Rect`, the main property where `Auto` is relevant is `margin` where `Auto` means "take up remaining available space" and `Undefined` resolves to zero. Currently Taffy and (bevy_ui) also accept `Auto` for `padding` and `border` where it is no different to `Undefined`. Taffy 0.3 (unreleased) is more strict about this and introduces a new type with no `Auto` variant for `padding` and `border`. I'm unsure as whether `bevy_ui` will follow Taffy in this regard.

Reply:
> I'm unsure as whether bevy_ui will follow Taffy in this regard.

I would be strongly in favour of doing so.

Reply:
At some point we should probably have that in the doc comments as well, of the respective components.